ludwig/experiment.py

```diff
@@ -308,52 +312,31 @@ def cli(sys_argv):
     # Data parameters
     # ---------------
     parser.add_argument(
-        '--data_csv',
-        help='input data CSV file. If it has a split column, it will be used '
-             'for splitting (0: train, 1: validation, 2: test), otherwise the '
-             'dataset will be randomly split'
+        '--dataset',
```

Comment:
We should probably add a test for this in `test_cli.py` at some point.

Reply:
Yes test_cli should test all of them, otherwise we will keep forgetting changing the cli parameters as they don't ger refactored automatically.